### DISCUSSION TITLE
Duplicate entries in grafana infinity datasource

### DISCUSSION THREAD

**Discussion by rameshsneha** [grafana/grafana-infinity-datasource#102] [Q&A] [answered]
Hi Team,

I am working on a project with grafana infinity datasource . When ever i do a refresh of panel , I see same data gets appended again into the table .

Any insight on this is much appreciated .

Thanks

**Comment by yesoreyeram** [comment, marked as answer]
Can you help me understand what version of panel/grafana you are using? If possible with dashboards/panel json.

**Reply by rameshsneha** [reply]
We found the issue , issue was with our proxy class . Solved now

Thanks for your time .

**Reply by yesoreyeram** [reply]
Glad you found the issue.
